src/main/java/net/minestom/server/network/packet/client/login/EncryptionResponsePacket.java

```diff
@@ -23,10 +23,7 @@
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Why wildcard import?

Reply:
:shrug:
IntelliJ automatically inserted it. I suppose it trivially saves a few lines of code, and it only would ever hurt things in the off chance there's a name collision.